wrappers/swift/EventProperties.swift

```diff
@@ -0,0 +1,285 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/**
+Represents Event's properties.
+*/
+public class EventProperties {
```

Comment:
Please check what's then naming convention we use for Swift in APEX/Apple and adhere

Reply:
Found it, it's the 3 letters of the project, in this case it might be SDW

Reply:
thanks for sharing it. is there a doc/wiki for the conventions?

Reply:
Looking at other examples in Office codebase, it seems Swift naming guidance is different from Obj-C, so you might leave class names as you have it now